lib/hash.h

```diff
@@ -103,7 +103,7 @@ struct hash {
  *
  * hash_cmp
  *    comparison function used for resolving collisions; when called with two
- *    data items, should return nonzero if the two items are equal and 0
+ *    data items, should return true if the two items are equal and false
```

Comment:
the same doc comment here as well https://github.com/FRRouting/frr/blob/c8aad9c3a4b3e6d6d85fcedaab80cc271015b1f9/lib/hash.h#L140